Doc/library/sqlite3.rst

```diff
@@ -1223,6 +1250,42 @@ Connection objects
 
       .. versionadded:: 3.11
 
+   .. attribute:: autocommit
+
+      This attribute controls :pep:`249`-compliant transaction behaviour.
+      *autocommit* has three allowed values:
+
+      * ``False``: select :pep:`249`-compliant transaction behaviour,
+        implying that :mod:`!sqlite3` ensures a transaction is always open.
+        Use :meth:`~Connection.commit` and :meth:`~Connection.rollback` to
+        close transactions.
+
+        This is the recommended value of *autocommit*.
+
+      * ``True``: Use SQLite's `autocommit mode`_.
+        :meth:`~Connection.commit` and :meth:`~Connection.rollback`
+        have no effect in this mode.
```

Comment:
Does using either of these affect the behavior of `isolation_level`, or mean that it is ignored? You probably want to mention that if so, either in a note/etc. if they both have the same effect (e.g. ignoring it), or separately in each one if they affect it differently.